src/3rdparty/webkit/Source/JavaScriptCore/heap/MarkedBlock.h

```diff
@@ -77,7 +77,7 @@ namespace JSC {
         template <typename Functor> void forEach(Functor&);
 
     private:
-        static const size_t blockSize = 16 * KB;
+        static const size_t blockSize = 64 * KB;
```

Comment:
If you look at the [commit in WebKit](https://trac.webkit.org/changeset/105442/webkit#file22), it was part of a larger change which implemented a new allocator. I'm not sure whether it would be cause some other regression, so please re-work this by fixing `PageAllocationAligned.cpp` -- I believe [the link referenced here](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3540#issuecomment-313180228) can be the start of a better solution.

Reply:
Possibly adding
if (alignment > pagesize)
       alignmentDelta = alignment - pagesize; https://github.com/wkhtmltopdf/qt/blob/wk_4.8.7/src/3rdparty/webkit/Source/JavaScriptCore/wtf/PageAllocationAligned.cpp#L53 is better